### PR TITLE
build from source documentation updates

### PR DESCRIPTION
**Pull request by daineAMD**

### CODE REVIEW

**Review by TorreZuk — APPROVED** [review]
Reads fine to me.  Assume you tested rmake -d as I haven't lately.

**Review by randyh62 — APPROVED** [review]
Looks good.

**Review by randyh62 — APPROVED** [review]
Looks good.

File: docs/install.rst [hunk 1]
```diff
@@ -14,8 +14,8 @@ Prerequisites
 * If using the rocBLAS backend on an AMD machine:
 
   * A ROCm enabled platform. Find more information on the :doc:`System requirements (Linux) <rocm-install-on-linux:reference/system-requirements>` page, or the :doc:`System requirements (Windows) <rocm-install-on-windows:reference/system-requirements>` page.
-  * A compatible version of rocBLAS
-  * A compatible version of rocSOLVER for full functionality
+  * A compatible version of rocBLAS.
+  * A compatible version of rocSOLVER and it's dependencies rocSPARSE and rocPRIM for full functionality.
```
Comment (LisaDelaney) [inline docs/install.rst]:
```suggestion
  * A compatible version of rocSOLVER and its dependencies, rocSPARSE and rocPRIM, for full functionality.
```

File: docs/install.rst [hunk 2]
```diff
@@ -151,13 +161,17 @@ in the table below.
 Dependencies For Building Library
```
Comment (LisaDelaney) [inline docs/install.rst]:
```suggestion
Dependencies for building library
```

File: docs/install.rst [hunk 3]
```diff
@@ -76,22 +78,32 @@ in the table below.
...
 
-=============== ====================================================
-executable name description
-=============== ====================================================
-hipblas-test    runs Google Tests to test the library
-hipblas-bench   executable to benchmark or test individual functions
-example-sscal   example C code calling hipblas_sscal function
-=============== ====================================================
+================= ====================================================
+executable name   description
```
Comment (LisaDelaney) [inline docs/install.rst]:
```suggestion
Executable name   Description
```

File: docs/install.rst (hunk 2 above)
Comment (Rmalavally) [reply docs/install.rst]: How about 'Dependencies for building library'

Prepositions must be in lowercase.

@LisaDelaney

Comment (daineAMD) [reply docs/install.rst]: Thanks for the reviews. I've added the suggested changes, including Roopa's here, in the latest commit.

**Pull request merged**